### DISCUSSION TITLE
Trying to use the deskew function from Wand and getting the below error.

### DISCUSSION THREAD

**Discussion by uday-felix** [ImageMagick/ImageMagick#4929] [Help]
I was facing this issue after editing the policy.xml file as well. 

Note: I have 1000+ files running in production and goes through the deskew function in a multithread. How do I deal with this?

Error:
`image_process.image_utils:115 — ERROR — Exception:MagickReadImage returns false, but did not raise ImageMagick exception. 
This can occur when a delegate is missing, or returns EXIT_SUCCESS without generating a raster`

**Comment by snibgo** [comment]
> MagickReadImage returns false

What is your input file?

>  This can occur when a delegate is missing, or...

Perhaps you are trying to read a file that isn't an image file, or it is but you don't have a delegate for that format, or ...

I don't understand your question about "deskew function in a multithread".

**Reply by uday-felix** [reply]
@snibgo,

My input files are are all in images that got converted to byte_array and sent to the deskew function. Below is my code for reference.
`    try:
        with skew(blob=img_png) as img_png_1:            
            img_png_1.deskew(0.5 * img_png_1.quantum_range)            
            img_png = bytes(img_png_1.make_blob())            
        log.info("Corrected the Skewness for %s", file_name)
    except Exception as e:`

the input is not one file at a time, its in a multithread and runs multiple files in a thread.

**Reply by uday-felix** [reply]
@snibgo Any update ?

**Comment by snibgo** [comment]
You have shown some code. I don't even know what language that is written in. I don't know what part of that code calls the function MagickReadImage(). I don't know what your input image is.

Sorry, I don't see how I can help.
